helm/robusta/values.yaml

```diff
@@ -35,6 +35,11 @@ globalConfig:
   custom_annotations: []
   custom_severity_map: {}
 
+# k8s service config
+service:
```

Comment:
Do you think we can move this into the `runner` section?
Since these annotations are for the `runner` service, I think it might belong there

Reply:
`runner` section indeed seems to be a better place for that.
Thanks for taking a look!